src/test/resources/expected.tsv

```diff
@@ -0,0 +1,7 @@
+TEST1	AD	Mental_retardation__autosomal_dominant_42:AD
+TEST2	AD	Mental_retardation__autosomal_dominant_42:AD
+TEST3	AD/SM	{Epilepsy__juvenile_myoclonic__susceptibility_to}:AD/SM,{Epilepsy__idiopathic_generalized__10}:AD,{Epilepsy__generalized__with_febrile_seizures_plus__type_5__susceptibility_to}:AD
```

Comment:
are the `{` and `}` intended?

Reply:
yes, they occur in omim, so I want to test if the regular expression can cope with them

Reply:
How about copying parts of this FAQ entry https://www.omim.org/help/faq#1_6 into our doc?
Are we dealing with the `[` / `]` case already?

Reply:
currently the string is used as it is in the input, so the [] and {} will be in the output as well.